goformation.go

```diff
@@ -14,6 +14,13 @@ import (
 // Open and parse a AWS CloudFormation template from file.
 // Works with either JSON or YAML formatted templates.
 func Open(filename string) (*cloudformation.Template, error) {
+	return OpenWithOptions(filename, nil)
+}
+
+// OpenWithOptions opens and parse a AWS CloudFormation template from file.
+// Works with either JSON or YAML formatted templates.
+// Parsing can be tweaked via the specified options.
+func OpenWithOptions(filename string, options *intrinsics.ProcessorOptions) (*cloudformation.Template, error) {
```

Comment:
`options` seems to be not used in this function.

Reply:
You're correct. I'll make the changes and update the PR.